### ISSUE TITLE
dayClick incorrectly triggered on scroll with touch devices

### ISSUE THREAD

**Issue by mitomm** [fullcalendar/fullcalendar#3152]
It seems that dayClick is triggered when the calendar is scrolled (by drag it) on touch devices (at least on iOS).

I reproduced the bug here: http://jsfiddle.net/1x8L2cjr/8/

Thanks for your help

**Comment by dancintony** [comment]
I noticed the same thing. I was able to work around it with the following trick (though I'd prefer dayClick not fire during a drag):

`var dragging = false;`

`$("body").on("touchmove", function(){
    dragging = true;
});`
`$("body").on("touchstart", function(){
    dragging = false;
});`

Then in your dayClick, check to see if `dragging` is true and do an early return.

**Comment by krasnoukhov** [comment]
Here is my workaround for it:

```js
dayClick: (date, jsEvent, view, resource) => {
  // [FIXME] Workaround for https://github.com/fullcalendar/fullcalendar/issues/3152
  if (jsEvent.originalEvent.type === "scroll") {
    return; 
  }

  // ...
}
```

**Comment by rahulrulez** [comment]
I can confirm this, we had to fallback to old library because of this. We updated the plugin one day before going live and it broke our web app on touch devices..

**Comment by arshaw** [comment]
thank you for this! will get this fixed

**Comment by arshaw** [comment]
can confirm. always accompanied with this:

![screen shot 2016-05-18 at 9 11 48 pm](https://cloud.githubusercontent.com/assets/56520/15382256/3794e926-1d3d-11e6-8071-456905421947.png)


probably a recently-introduced chrome behavioral change

**Comment by arshaw** [comment]
this is fixed in [v2.7.2](https://github.com/fullcalendar/fullcalendar/releases/tag/v2.7.2). could you please verify?
